pyiron_base/job/generic.py

```diff
@@ -965,7 +958,11 @@ def run_if_scheduler(self):
             raise ValueError("run_queue.sh crashed")
         s.logger.debug("submitted %s", self.job_name)
         self._logger.debug("job status: %s", self.status)
-        return que_id
+        self._logger.info(
+            "{}, status: {}, submitted: queue id {}".format(
+                self.job_info_str, self.status, que_id
+            )
+        )
```

Comment:
We should still return the queue id and I thought this information, is logged already.  Looks good otherwise, though.

Reply:
I noticed that we only return the queue id to print it in the log as the rest ist already handled in the lines above, so I simply moved the logging part to the same place and then there is no need to return the queue id any more.